ramls/bound-with-part.json

```diff
@@ -0,0 +1,29 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "description": "Records the relationship between a part of a bound-with (a holdings-record) and the bound-with as a whole (the circulatable item)",
+  "type": "object",
+  "properties": {
+    "id": {
+      "type": "string",
+      "description": "unique ID of the recorded bound-with relation; a UUID"
```

Comment:
This comment suggests that this is intended to be a UUID, could it be worth including a constraint for this property that includes this in the machine readable part of the schema?

Reply:
Yes, that would be tidy. Done.